File: 1_problem/JAVA_code/Main1.java
public class Main1 {

	public static void main(String[] args) {
		int[] arr = {23, 6, 34, 2, 4, 89, 5};
		System.out.println("Smallest element is: " + small_in(arr));
	}

	static int small_in(int[] arr) {
		int min = arr[0];
		for (int i = 0; i < arr.length; i++) {
			if (min > arr[i])
				min = arr[i];
		}
		return min;
	}
}

File: 1_problem/CPP_code/Main1.cpp
#include<bits/stdc++.h>
using namespace std;

int small_in(int arr[], int n) {
	int min = arr[0];
	for (int i = 1; i < n; i++) {
		if (min > arr[i])
			min = arr[i];
	}
	return min;
}

int main() {
	int arr1[] = {23, 6, 34, 2, 4, 89, 5};
	int n = 7;
	cout << "Smallest variable is: " << small_in(arr1, n) << endl;
}

